tests/test_e2e.py

```diff
@@ -0,0 +1,36 @@
+import json
+import tempfile
+import time
+from pathlib import Path
+
+import pytest
+
+from neptune_tensorflow_keras.impl import NeptuneCallback
+
+
+try:
+    # neptune-client=0.9.0+ package structure
+    from neptune.new import init_run
+except ImportError:
```

Comment:
Are we planning to run this test with older version of the client?

Reply:
We use this template in other repositories, so I assume that we want to be able to do this.